githooks/pre-push

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+echo "Running pre-push git hook: $0"
+./ci/check_fmt.sh
```

Comment:
Can you add a comment here explaining why you aren't piping the output of this one?